build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -112,7 +111,7 @@
   <Target Name="SetupRestoreTestsInputs"
           DependsOnTargets="PrepareTests;" >
     <ItemGroup>
-      <RestoreTestsInputs Include="$(TestDirectory)/**/*.csproj"
+      <RestoreTestsInputs Include="$(TestDirectory)/Microsoft.DotNet.Cli.Tests.sln"
```

Comment:
Well, I wanted to do as little as possible to disrupt the current state. Lemme try without it and see where we end up...